Upcoming-iCal-Events.widget/main.coffee

```diff
@@ -30,6 +58,7 @@ style: """
     #head
         font-weight: bold
         font-size 20px
+        padding-bottom 6px
```

Comment:
nit; i've tried to make the spacing as small as possible to allow for smaller screensizes and people who want to use other widgets above/below this one. Would prefer this to be added by people if they want to adjust the spacing vs being the default.

Reply:
Removed.